dist/mapbox-gl.css

```diff
@@ -267,3 +274,11 @@
         display:none;
     }
 }
+.pseudo-fullscreen {
```

Comment:
This doesn't seem to be used (and we shouldn't need to support "pseudo-fullscreen" mode, as all browsers supported by gl-js should support the HTML5 fullscreen API).